test/unittest/create_test.sh

```diff
@@ -0,0 +1,62 @@
+#! /bin/bash
+
+# First argument needs to be the name of the script.
+
+usage_message="Usage: ./create_test.sh [--dir directory] my_test"
```

Comment:
Instead of --dir directory I think we instead move toward modules. So `-m module` seems like a better fit to me. That will then be the implied folder that the test is placed in.

Reply:
Done, lmk if it looks better.

Reply:
Looks better.